.travis.yml

```diff
@@ -5,7 +5,6 @@ language: python
 
 env:
   matrix:
-    - PYTHON_VERSION="2.6" NUMPY_VERSION="1.6"
```

Comment:
Yep good point, should be quick with the scripts, also add py27-np16 to the list of conda envs in the main script regenerating the data for all the envs.